From: Robert Rostohar <Robert.Rostohar@arm.com>
Date: Mon, 4 Jan 2021 19:26:30 +0100
Subject: [PATCH] Update tfm_cmake_defaults.py script

- remove non-existent file in TF-M root detection
- convert back-slash '\' to slash '/' when rendering configuration
---
 tools/documentation/tfm_cmake_defaults.py | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/tools/documentation/tfm_cmake_defaults.py b/tools/documentation/tfm_cmake_defaults.py
index 5fb7b86f..6c33db21 100644
--- a/tools/documentation/tfm_cmake_defaults.py
+++ b/tools/documentation/tfm_cmake_defaults.py
@@ -32,8 +32,7 @@ tfm_def_build_doxygen = True
 
 def find_tfm_root(start_dir=os.path.dirname(os.path.abspath(__file__)),
                   target_files=["license.rst",
-                                "dco.txt",
-                                "CommonConfig.cmake"],
+                                "dco.txt"],
                   max_depth=5):
     """ Method which attempts to find the root of the project
     by traversing parent directoried and attempts to located each of the
@@ -77,7 +76,7 @@ def render_cmake_file(config_map, in_file, out_file):
 
     # Render all config entires included in the map
     for k, v in config_map.items():
-        v = v.replace("\\", "\\\\")
+        v = v.replace("\\", "/")
         _data = re.sub(r'@%s@' % k, r'%s' % v, _data)
 
     # Set all remaining entries to blank
-- 
